advisor/index.go

```diff
@@ -29,6 +29,11 @@ import (
 	"vitess.io/vitess/go/vt/sqlparser"
 )
 
+const (
+	// https://dev.mysql.com/doc/refman/8.0/en/identifiers.html
```

Comment:
comment format should be
// IndexNameMaxLength Ref. https://dev.mysql.com/doc/refman/8.0/en/identifiers.html